### PR TITLE
Update research.rst

### PR DESCRIPTION
**Pull request by engsbk**
Added a new paper.

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -224,6 +224,7 @@ DeepONet
 #. S\. Cai, Z. Wang, L. Lu, T. Zaki, & G. Karniadakis. `DeepM&Mnet: Inferring the electroconvection multiphysics fields based on operator approximation by neural networks <https://doi.org/10.1016/j.jcp.2021.110296>`_. *Journal of Computational Physics*, 436, 110296, 2021.
 #. L\. Lu, P. Jin, G. Pang, Z. Zhang, & G. Karniadakis. `Learning nonlinear operators via DeepONet based on the universal approximation theorem of operators <https://doi.org/10.1038/s42256-021-00302-5>`_. *Nature Machine Intelligence*, 3, 218--229, 2021.
 #. C\. Lin, Z. Li, L. Lu, S. Cai, M. Maxey, & G. Karniadakis. `Operator learning for predicting multiscale bubble growth dynamics <https://doi.org/10.1063/5.0041203>`_. *The Journal of Chemical Physics*, 154(10), 104118, 2021.
+#. Alkhadhr, S., & Almekkawy, M. (2023). `Wave Equation Modeling via Physics-Informed Neural Networks: Models of Soft and Hard Constraints for Initial and Boundary Conditions<https://www.mdpi.com/1424-8220/23/5/2792>`_. Sensors, 23(5), 2792.
```
Comment (lululxvi) [inline docs/user/research.rst:227]: Wrong location. It is PINN, not DeepONet.

Comment (engsbk) [reply docs/user/research.rst]: Corrected.

File: docs/user/research.rst [hunk 2]
```diff
@@ -115,6 +115,8 @@ Here is a list of research papers that used DeepXDE. If you would like your pape
 PINN
 ----
 
+#. Alkhadhr, S., & Almekkawy, M. (2023). `Wave Equation Modeling via Physics-Informed Neural Networks: Models of Soft and Hard Constraints for Initial and Boundary Conditions<https://www.mdpi.com/1424-8220/23/5/2792>`_. Sensors, 23(5), 2792.
+
```
Comment (lululxvi) [inline docs/user/research.rst:119]: Delete this empty line.

Comment (engsbk) [reply docs/user/research.rst]: Deleted.

File: docs/user/research.rst [hunk 3]
```diff
@@ -115,6 +115,7 @@ Here is a list of research papers that used DeepXDE. If you would like your pape
 PINN
 ----
 
+#. Alkhadhr, S., & Almekkawy, M. (2023). `Wave Equation Modeling via Physics-Informed Neural Networks: Models of Soft and Hard Constraints for Initial and Boundary Conditions<https://www.mdpi.com/1424-8220/23/5/2792>`_. Sensors, 23(5), 2792.
```
Comment (lululxvi) [inline docs/user/research.rst:118]: The format of author names should be modified. See other examples.

File: docs/user/research.rst [hunk 4]
```diff
@@ -115,6 +115,7 @@ Here is a list of research papers that used DeepXDE. If you would like your pape
 PINN
 ----
 
+#. S\. Alkhadhr, & M\. Almekkawy. `Wave Equation Modeling via Physics-Informed Neural Networks: Models of Soft and Hard Constraints for Initial and Boundary Conditions<https://www.mdpi.com/1424-8220/23/5/2792>`_. *Sensors*, 23(5), 2792, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:118]: `M\. Almekkawy`

Comment (lululxvi) [inline docs/user/research.rst:118]: Title no capital.

File: docs/user/research.rst [hunk 5]
```diff
@@ -115,6 +115,7 @@ Here is a list of research papers that used DeepXDE. If you would like your pape
 PINN
 ----
 
+#. S\. Alkhadhr, & M\. Almekkawy. `Wave equation modeling via physics-informed neural networks: models of soft and hard constraints for initial and boundary conditions<https://www.mdpi.com/1424-8220/23/5/2792>`_. *Sensors*, 23(5), 2792, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:118]: - `M. Almekkawy`
- `conditions <https`

**Comment by lululxvi** [comment]
Add Kuwait University to the list of universities

**Pull request merged**